inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/es/ElasticsearchApi.java

```diff
@@ -76,7 +93,157 @@ public boolean indexExists(String indexName) throws IOException {
         return getEsClient().indices().exists(getIndexRequest, RequestOptions.DEFAULT);
     }
 
+    /**
+     * Create index
+     *
+     * @param indexName The index name of elasticsearch
+     * @return void
+     * @throws IOException The exception may throws
+     */
+    public void createIndex(String indexName) throws IOException {
+        CreateIndexRequest createIndexRequest = new CreateIndexRequest(indexName);
+
+        CreateIndexResponse createIndexResponse = getEsClient().indices()
+                .create(createIndexRequest, RequestOptions.DEFAULT);
+        LOG.info("create es index: {}", createIndexResponse.isAcknowledged());
```

Comment:
Maybe adding more log info is good to locate problem.

Reply:
Add indexName to locate problem